src/stores/watchlistTableStore.ts

```
@@ -1,38 +1,62 @@
-        return sCoinId !== coinId;
-      });
-      watchListedTokens.value = filteredWatchListArray
+    if (isAlreadyListed) {
+      targetList.value = targetList.value.filter(item => item !== id)
+    } else {
+      targetList.value = [...targetList.value, id]
     }
+  }
+
+  const isWatchListed = (coinId: string) => {
+    const isStoredToken = watchListedTokens.value.find((sCoinId: string) => {
```

Biome (2.3.14)

[error] This callback passed to find() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)